### DISCUSSION TITLE
how to run "waifu2x" with zluda ?

### DISCUSSION THREAD

**Discussion by patientx** [vosen/ZLUDA#204] [Q&A]
It is mentioned in the read.me that it supports waifu2x, but how exactly ? Not through gui I presume , and I tried running it with 

"call d:\temel\ZLUDA\zluda.exe -- waifu2x-caffe-cui.exe  -i p.jpg"

but it gives 

F0415 18:31:40.179004  1608 math_functions.cu:68] Check failed: status == CUBLAS_STATUS_SUCCESS (15 vs. 0)  CUBLAS_STATUS_NOT_SUPPORTED
*** Check failure stack trace: ***

errors.

**Comment by vosen** [comment]
README talks about original waifu2x (https://github.com/nagadomi/waifu2x), you seem to be using waifu2x-caffe (https://github.com/lltcggie/waifu2x-caffe). This is a different project with different code base. It has not been tried with ZLUDA
